src/configurations.jl

```diff
@@ -960,7 +960,7 @@ julia> spin_configurations(c"1s2")
 
 julia> spin_configurations(c"1s2"s)
 1-element Array{Configuration{SpinOrbital{Orbital{Int64},Tuple{Int64,HalfIntegers.Half{Int64}}}},1}:
- 1s²
```

Comment:
Is this change correct?

Reply:
It's correct as long as #71 is not resolved.